functions.php

```diff
@@ -442,9 +442,14 @@ function odm_return_all_topics( $query ) {
 
 function odm_search_pre_get_posts($query)
 {
+  if(get_post_type()){
+    $post_type = $query->query['post_type'][0];
```

Comment:
Did it affect to the search function in your side? I tested in my DE, it does not.